.travis.yml

```diff
@@ -56,5 +56,14 @@ deploy:
       repo: watson-developer-cloud/java-sdk
       jdk: oraclejdk7
 
+  - provider: releases
+    api_key:
+      secure: NhwGz9ppK1YM0M9Pr15b6fOHPpN09QrLvdK4z1PTRxAt+oVpzl3xM/CpGr+iRXT3aV+8q0xLtb4rjxdmgV7aDFY5WEcKkdHAlENpLk/ZVCswnl/MakNbupFbiml0aLcNRe90qvuMILjwSfzYU6BlxZ/W1Xz1eqwtQ1oP/A92MZM=
```

Comment:
Use `$GITHUB_TOKEN` so that we can control this value on the Travis settings page. I don't like doing commits to change a secret.